bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java

```diff
@@ -306,6 +305,16 @@ public void onSuccess(UcpRequest request) {
         }
 
         assertTrue(success.get());
+        UcpRequest close = ep.closeNonBlockingForce();
+
+        while (!close.isCompleted()) {
```

Comment:
why not progressRequest?

Reply:
Progress happens in progress thread: https://github.com/openucx/ucx/blob/e5f55cd61aa167e1ce96e2350c62303499c0136e/bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java#L273